react/features/base/react-native/AudioMode.js

```diff
@@ -0,0 +1,20 @@
+import { NativeModules } from 'react-native';
+import { Platform } from '../react';
+
+let AudioMode;
+
+if (Platform.OS === 'ios') {
+    AudioMode = NativeModules.AudioMode;
+} else {
+    // TODO(saghul): Implement for Android
+    AudioMode = {
+        DEFAULT: 0,
+        AUDIO_CALL: 1,
+        VIDEO_CALL: 2,
+        setMode() {
+            return Promise.resolve(null);
+        }
+    };
+}
+
+export default AudioMode;
```

Comment:
`AudioMode` is an abstraction (to me) that I don't see being used outside its feature. Unlike `POSIX` which started with the idea to be a collection of POSIX functions and, thus, likely belongs to features/base.